### ISSUE TITLE
[Web] Lighthouse budgets in CI

### ISSUE THREAD

**Issue by Sudo-Ivan** [melovian-hq/Melovian-Web#5]
lighthouserc.json exists. Add assertions so perf/a11y regressions fail CI.
